cactusbot/handlers/events.py

```diff
@@ -23,8 +26,20 @@ def on_follow_return():
 
         if packet.success:
             if self.cache_follows:
-                if not(CacheUtils.in_cache_followers(packet.user)):
-                    CacheUtils.cache_followers_add(packet.user)
+                cache = CacheUtils("caches/followers.json")
+                if cache.in_cache(packet.user):
+                    if self.cache_follows_time > 0:
+                        configtime = int(self.cache_follows_time * 60)
```

Comment:
Because it looks like we only need the "`* 60` version" of `cache_follows_time` (only need `180`, never `3`), might as well move the `* 60` to `__init__`.

(Also, for future reference, there's no need to `int()` here. :smile: )